<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="infowidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>351</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QPushButton" name="registerProfile">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Save and continue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registerFirstInput">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registerLastInput">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registerWeightInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Your weight (kg)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registerHeightInput">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Your height (cm)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="startwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QPushButton" name="createProfileButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Create profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="enterButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>340</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RaDoTech</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="homewidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QPushButton" name="homeMeasureButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Measure now</string>
     </property>
    </widget>
    <widget class="QLabel" name="homeNameLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NAME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="homeProfilesButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Profiles</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeHistoryButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>History</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeLogoutButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Log out </string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifyProfileButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>410</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modify profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteProfileButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>410</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete profile</string>
     </property>
    </widget>
    <widget class="QPushButton" name="homeGraphButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Graph</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="profilewidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Profiles</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profile1Button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profile2Button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profile3Button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profile4Button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>320</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="profile5Button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>390</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Profile5</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="battery_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>-10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Battery life: 100</string>
    </property>
   </widget>
   <widget class="QWidget" name="measurewidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QPushButton" name="measureButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>MEASURE</string>
     </property>
    </widget>
    <widget class="QLabel" name="currBodyLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>BODY PART MEASURING: FEET</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="bodyMeasurementLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current Point:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Skin Contact:</string>
     </property>
    </widget>
    <widget class="QLabel" name="skinContactLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ApplyWater">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>APPLY WATER</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="resultswidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QLabel" name="energyLevel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Energy level:</string>
     </property>
    </widget>
    <widget class="QLabel" name="immuneSystem">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Immune system:</string>
     </property>
    </widget>
    <widget class="QLabel" name="metabolism">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Metabolism:</string>
     </property>
    </widget>
    <widget class="QLabel" name="psychoemotional">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Psycho-emotional state:</string>
     </property>
    </widget>
    <widget class="QLabel" name="musculoskeletal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Musculoskeletal system:</string>
     </property>
    </widget>
    <widget class="QLabel" name="average">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Average values:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resultsHomeButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>GO HOME</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resultsRecButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>460</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>RECS</string>
     </property>
    </widget>
    <widget class="QLabel" name="leftLung">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Left Lung:</string>
     </property>
    </widget>
    <widget class="QLabel" name="rightLung">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Right Lung:</string>
     </property>
    </widget>
    <widget class="QLabel" name="heart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Heart:</string>
     </property>
    </widget>
    <widget class="QLabel" name="stomach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stomach:</string>
     </property>
    </widget>
    <widget class="QLabel" name="largeIntestine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Large Intestine:</string>
     </property>
    </widget>
    <widget class="QLabel" name="smallIntestine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Small Intestine:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="recommendationwidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>391</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Recommendations:</string>
     </property>
    </widget>
    <widget class="QLabel" name="recommendations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>341</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="recBackButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>GO BACK</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="statswidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QPushButton" name="graphHomeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GO HOME</string>
     </property>
    </widget>
    <widget class="QWidget" name="graphwidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>361</width>
       <height>491</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="historywidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QWidget" name="historylistwidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>361</width>
       <height>481</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="historyHomeButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>83</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>GO HOME</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="exitwidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>361</width>
      <height>521</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>291</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Device shutting off in:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="countdownLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>24</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>infowidget</zorder>
   <zorder>startwidget</zorder>
   <zorder>homewidget</zorder>
   <zorder>profilewidget</zorder>
   <zorder>battery_label</zorder>
   <zorder>measurewidget</zorder>
   <zorder>resultswidget</zorder>
   <zorder>statswidget</zorder>
   <zorder>historywidget</zorder>
   <zorder>exitwidget</zorder>
   <zorder>recommendationwidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
